src/screen_capture/ScreenCapture.cpp

```diff
@@ -0,0 +1,226 @@
+#include <tuple>
+#include "ScreenCapture.h"
+
+namespace OutputCapture
+{
+    ScreenCapture::ScreenCapture()
+    {}
+
+    ScreenCapture::~ScreenCapture()
+    {
+        DeleteInstance();
+    }
+
+    ScreenCapture *OutputCapture::ScreenCapture::pInstance = nullptr;
+
+    bool ScreenCapture::CaptureFullScreen(std::string outputFileName) const

```

Comment:
same com as above about your bool

Reply:
Same, do you think it'd be better to throw an exception instead of returning a boolean ?